Fastfile

```diff
@@ -25,20 +25,34 @@ default_platform :ios
     icon_overlay(version: get_version_number)
     set_build_number()
     update_info_plist
-    gym(use_legacy_build_api: true)
-    custom_notes = ENV['CUSTOM_HOCKEY_RELEASE_NOTES'] || ""
+    build_with_gym()
+    upload_to_hockey()
+  end
+
+  def build_with_gym()
+    provisioning_profile_name = ENV['TAB_PROVISIONING_PROFILE']
+    if provisioning_profile_name != nil
+      xcconfig_filename = Dir.pwd + "/TAB.release.xcconfig"
+      File.write(xcconfig_filename, "PROVISIONING_PROFILE_SPECIFIER = #{provisioning_profile_name}\n")
```

Comment:
So does the file automatically inherit as it is used as an "additional xcconfig file"?